source/_integrations/watergate.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: Watergate
+description: Instructions on how to integrate Watergate within Home Assistant.
+ha_category:
+  - Sensor
+  - Valve
+  - Water management
+ha_release: '2025.1'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@adam-the-hero'
+ha_domain: watergate
+ha_platforms:
+  - valve
+  - sensor
+---
+
+The **Watergate** {% term integration %} integrates your Watergate Devices (currently Sonic) to your Home Assistant.
+With this integration you are able to:
+
+- control your valve
+- monitor telemetry
+- gather auto shut off reports
+- check diagnostics of the device
+
+## Prerequisites
+
+- You need to have Sonic device
+- Local API feature must be enabled in application
+- Network Traffic between Home Assistant Server and Sonic must be unlocked (Server uses HTTP protocol on 80 port).
+
+{% important %}
+Go to Watergate application and enable Local API in specific device's settings.
```

Comment:
This is a duplicate of the second point?

Reply:
Yes it is. I've removed it I think there is no point to write 101 instructions for that.